### PR TITLE
CI fix

### LINKED ISSUE #1165: Can help to publish 2.2.4.1 version in synology docker (current is 2.2.4)?
### Version, approach of installation, operating system
sysnology dsm7.2
1. What version of v2rayA are you using?
2.2.4
2. What approach of installation do you install V2rayA?
docker
3. What is your operating system?
dsm7.2

### Description

the latest version is 2.2.4 in which exits GFW list issue, as we know this is fixed in 2.2.4.1 version here, can help to publish 2.2.4.1 version in docker? Thanks!

Snipaste_2023-11-10_22-27-49

### PR DESCRIPTION
**Pull request by Victor2333**
#1165
Move GUI building step from docker to ubuntu root job.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -4,19 +4,20 @@ ADD .git ./.git
 RUN git describe --abbrev=0 --tags | tee ./version
 
 
-FROM node:lts-alpine AS builder-web
-ADD gui /build/gui
-WORKDIR /build/gui
-RUN echo "network-timeout 600000" >> .yarnrc
+# FROM node:lts-alpine AS builder-web
```
Comment (Gaojianli) [inline Dockerfile:7]: After this changed, this Dockerfile is not usable for those who want build docker images by themselves

Comment (Victor2333) [reply Dockerfile:7]: According https://github.com/docker/setup-qemu-action/issues/110
Git hub action will always timeout for qemu armv7 which is unnecessary.
And yes. Those who want build docker image they need build frontend first and then do docker build.
I will create a docker dev file to support local build.

Comment (Gaojianli) [reply Dockerfile:7]: I am afraid I can't agree with you. 
>It cause build always failed in git action https://github.com/v2rayA/v2rayA/actions/runs/6764494537/job/18563292110

In this log, I see the build was cancelled by something, and the GUI built succeed.
image

Comment (Victor2333) [reply Dockerfile:7]: image
image
Build failed due to timeout. It takes 6 hours on arm/v7 platform doing yarn and yarn build.

**Pull request closed without merging**